camus-shopify/script/update_schedule

```diff
@@ -48,8 +48,8 @@ def main():
     camus_jar = get_camus_version(repo_root)
     azkaban_auth = json.load(open(os.path.join(repo_root, 'camus-shopify', 'azkaban.json')))
 
-    command = 'bash -c "java -Xms1G -Xmx2G -DHADOOP_USER_NAME=deploy '
-    command += '-Dlog4j.configuration=file:/u/apps/camus/shared/log4j.xml '
+    command = 'bash -c "curl --compressed -H "Accept: application/json" -X GET "http://hadoop-rm.chi.shopify.com:8088/ws/v1/cluster/apps?states=RUNNING,ACCEPTED" | grep -v Camus && echo \'Camus is not running\' && '
+    command += 'java -Xms1G -Xmx2G -DHADOOP_USER_NAME=deploy -Dlog4j.configuration=file:/u/apps/camus/shared/log4j.xml '
     command += '-cp $(hadoop classpath):/u/apps/camus/current/camus-shopify-0.1.0-shopify1.jar:/etc/camus:/etc/hadoop/conf '
```

Comment:
You don't need to append `/etc/hadoop/conf` as that is already returned by `hadoop classpath`.